.github/workflows/manage-github-runners.yml

```
@@ -0,0 +1,110 @@
+  workflow_dispatch:
+    inputs:
+      mode:
+        type: choice
+        description: Choose runner mode
+        options:
+          - self-hosted
+          - github-hosted
+      imageTag:
+        type: string
+        required: false
+        description: Runner image tag (default: latest)
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)